scripts/distributeVersion.ts

```diff
@@ -0,0 +1,170 @@
+// Update TypeScript Version
+
+/// <reference path="../src/harness/external/node.d.ts" />
+"use strict";
+import * as fs from "fs";
+import * as path from "path";
+
+const versionFilePath = "package.json";
+
+interface FileInformation {
+    filePath: string;
+    ext: string;
+    content: string;
+}
+
+interface FormatInformation {
+    pattern: RegExp;
+    serialize: (version: string) => string;
+    nightlySerialize?: (nightlyVersion: string) => string;
+}
+
+/**
+ * A minimal description for a parsed package.json object.
+ */
+interface PackageJson {
+    name: string;
+    version: string;
+    keywords: string[];
+}
+
+/**
+ * A map between filePath and FormatInformation.
+ * A new file that need to be serialize and update version should be added in this map.
+ */
+const formats: {[idx: string]: FormatInformation} = {
+    "src/compiler/program.ts": {
+        pattern: /version\s*=\s*"[0-9]+\.[0-9]+\.[0-9]"/g,
+        serialize: (version: string) => {
+            return `version = "${version}"`;

```

Comment:
Can you make this `const version = ${version}` and the above `/const\s*version\s*=\s*"[0-9]+\.[0-9]+\.[0-9]"/g`?